conf/development.ini

```diff
@@ -3,4 +3,4 @@ use = call:via.app:create_app
 
 nginx_server: http://localhost:9083
 client_embed_url: http://localhost:5000/embed.js
-legacy_via_url: http://localhost:9080
+legacy_via_url: http://localhost:9083/html/v
```

Comment:
It works in all the same ways as Via for the most part, so our URL routing works for it already. 

Unlike original Via, the URL structure is a bit weird. It's:

 * `.../assets`
 * `.../collection/<url>`

Whereas Via has done some hackery to get it to be:

 * `.../assets`
 * `.../<url>`

The `html` part of the URL is so we don't clash with our own `/assets` which is routing through the app. The `v` in this case is just a short collection name. Doesn't really mean anything. `rewrite/assets` `rewrite/html` might be nicer.